hack/local-up-kubeedge.sh

```diff
@@ -191,6 +190,7 @@ function generate_streamserver_cert {
 
 cleanup
 
+source "${KUBEEDGE_ROOT}/hack/verify-golang.sh"
```

Comment:
verify golang looks more like a prerequisites instead, how about move this line to a function?

Reply:
Done, have moved the func to `hack/lib/golang`.